web_m2x_options/static/src/components/form.esm.js

```diff
@@ -25,6 +30,18 @@ Many2XAutocomplete.props = {
     fieldColorOptions: {type: Object, optional: true},
 };
 
+KanbanMany2OneAvatarUserField.props = {
+    ...KanbanMany2OneAvatarUserField.props,
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
+
+Many2OneAvatarUserField.props = {
+    ...Many2OneAvatarUserField.props,
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
+
```

Comment:
```suggestion
KanbanMany2OneAvatarField.props = {
    ...KanbanMany2OneAvatarField.props,
    fieldColor: {type: String, optional: true},
    fieldColorOptions: {type: Object, optional: true},
};
```

And change this line https://github.com/OCA/web/pull/2921/files#diff-ca954d78dad7b4338381958be4abd93f5fbea46f766a6af5e914a9bd74017737R9 by this 
```javascript
import {Many2OneAvatarField, KanbanMany2OneAvatarField} from "@web/views/fields/many2one_avatar/many2one_avatar_field";